modules/gateways/monero/createinvoice.php

```diff
@@ -39,15 +41,16 @@ function monero_payment_id(){
 $array_integrated_address = $monero_daemon->make_integrated_address($payment_id);
 $address = $array_integrated_address['integrated_address'];
 $uri  =  "monero:$address?amount=$amount_xmr";
-
+$url  = Capsule::table('tblconfiguration')->where('setting', 'SystemURL')->first();
+$system_url = rtrim($url->value, '/');  // Strips default trailing / if there
```

Comment:
This adds the system_url for the whmcs installation and removes the trailing / at the end for use later